ts/parts-3d/Column.ts

```diff
@@ -530,21 +572,51 @@ wrap(Series.prototype, 'alignDataLabel', function (
 wrap(H.StackItem.prototype, 'getStackBox', function (
     this: Highcharts.StackItem,
     proceed: Function,
-    chart: Highcharts.Chart
+    chart: Highcharts.Chart,
+    stackItem: Highcharts.StackItem,
+    x: number,
+    y: number,
+    xWidth: number,
+    h: number,
+    axis: Highcharts.Axis
 ): void { // #3946
     var stackBox = proceed.apply(this, [].slice.call(arguments, 1));
+    // Only do this for 3D column and iherited series.
+    if (chart.is3d() && stackItem.base) {
```

Comment:
How is this check ensuring that we have a 3D column or it's "iherited" (btw: should be inherited) series?
The check for "columness" is a bit lower in the code as far as I can tell.

Reply:
this check for columnSeries is just 10 lines below.

Reply:
... so I will move the comment to this place and correct a typo :)